owiddata/generate-owiddata-stats.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Get the most recent commit affecting this file and store the commit sha and date
+# See https://stackoverflow.com/questions/50194241/get-when-the-file-was-last-updated-from-a-github-repository/50204589
+#OWID_COMMIT_JSON=$(curl -sS "https://raw.githubusercontent.com/owid/covid-19-data/master/public/data/vaccinations/vaccinations.json")
```

Comment:
The EBM version is calling https://api.github.com as opposed to downloading the raw file.  Do you want me to try to fix this?  It may be hard for me to get the syntax correct via GitHub suggestions.  I could push to your branch to edit only these lines.

I do think we should track `OWID_COMMIT_SHA` and `OWID_COMMIT_DATE`.  These be helpful to figure out how often the external data source is updating and what version of the data we have.

Reply:
Thank you so much @agitter, that would be awesome! Please feel free to push to the branch, or let me know if I can make it easier in any way!